udata/templates/macros/metadata.html

```diff
@@ -47,6 +47,7 @@
 {% endif %}
 
 <meta name="check-urls" content="{{ config.LINKCHECKING_ENABLED|tojson }}" />
+<meta name="check-urls-cache-duration" content="{{ config.LINKCHECKING_CACHE_DURATION|tojson }}" />
```

Comment:
Is the `|tojson` filter still pertinent for numeric values (vs. boolean)?

Reply:
Agreed, does not transform anything